src/main/pages/che-7/overview/assembly_accessing-che-from-openshift-developer-perspective.adoc

```diff
@@ -8,15 +8,17 @@ folder: che-7/overview
 summary:
 ---
 
+:page-liquid:
+
```

Comment:
Parent context should be set.
```suggestion
 :parent-context-of-accessing-che-from-openshift-developer-perspective: {context}
 
```

Reply:
Also, reset the context at the end of the assembly with:
```
:context: {parent-context-of-accessing-che-from-openshift-developer-perspective}
```
(I can't add it as a suggestion.)